help/configure-emoticon-translations.md

```diff
@@ -21,7 +21,7 @@ automatically by Zulip.
 
 {settings_tab|display-settings}
 
-1. Under **Theme**, select **Convert emoticons before sending**.
+1. Under **Emoji**, toggle **Convert emoticons before sending**.
```

Comment:
Since now the instruction says "toggle, the heading for this section should be H2 `## Configure emoticon translations` instead of H3 `### Enable emoticon translations`.